scripts/jtsync/jtsync.rb

```diff
@@ -14,12 +14,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-require 'trello'
-require 'netrc'
-require 'optparse'
-require 'ostruct'
```

Comment:
ok if you have tested it, i am fine with it. i just tried it on `irb` and it wasn't required by default